### DISCUSSION TITLE
Test suite broken in current main?

### DISCUSSION THREAD

**Discussion by mwiebusch78** [ibis-project/ibis#9513] [General]
I would like to have a go at fixing some bugs and I followed the steps in the 'Contribute' section of the docs. However, I am getting errors when running the core tests (`pytest -m core`). It seems like pytest already fails at collecting the tests:

```
!!!! Interrupted: 19 errors during collection !!!!
```

The actual error messages are:

```
ERROR ibis/backends/tests/test_aggregation.py - DeprecationWarning: Accessing `ComputeError` from the top-level `polars` module is deprecated. Import it directly from the `polars.exceptions...
ERROR ibis/backends/tests/test_api.py - DeprecationWarning: Accessing `ComputeError` from the top-level `polars` module is deprecated. Import it directly from the `polars.exceptions...
ERROR ibis/backends/tests/test_array.py - DeprecationWarning: Accessing `ComputeError` from the top-level `polars` module is deprecated. Import it directly from the `polars.exceptions...
```

(And so on...) So they are just warnings, but for some reason pytest registers them as errors.

The polars version in my ibis-dev environment is 1.0.0

**Comment by mwiebusch78** [comment]
Manually downgrading polars to the last version before 1.0.0 fixed the issue.

**Reply by cpcloud** [reply]
Yep! Polars 1.0.0 isn't yet supported. A bunch of APIs changed and there's a fun new segfault when exporting decimal columns to pyarrow 👀
